packages/nvidia-container-toolkit/nvidia-container-toolkit-config.toml

```diff
@@ -0,0 +1,8 @@
+disable-require = false
+
+[nvidia-container-cli]
+root = "/"
+path = "/usr/bin/nvidia-container-cli"
+environment = []
+debug = "/var/log/nvidia-container-runtime-hook.log"
```

Comment:
Can we log to the journal instead, or make sure debug logging is off by default? I'm wary of logging to a file that we never truncate or rotate.